com.unity.perception/Documentation~/GroundTruth/KeypointLabeler.md

```diff
@@ -0,0 +1,91 @@
+# Keypoint Labeler
+
+The keypoint labeler captures the screen locations of specific points on labeled GameObjects. The typical use of this labeler is capturing human pose estimation data, but it can be used to capture points on any kind of object. The labeler uses a [keypoint template](#KeypointTemplate) which defines the keypoints to capture for the model and the skeletal connections between those keypoints. The positions of the keypoints are recorded in pixel coordinates.
```

Comment:
Since "Labeler" is a class name in our package, I have started capitalizing it in the docs as we move forward, similar to "Randomizer".

```suggestion
The Keypoint Labeler captures the screen locations of specific points on labeled GameObjects. The typical use of this Labeler is capturing human pose estimation data, but it can be used to capture points on any kind of object. The Labeler uses a [Keypoint Template](#KeypointTemplate) which defines the keypoints to capture for the model and the skeletal connections between those keypoints. The positions of the keypoints are recorded in pixel coordinates.
```